strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/LoggingManagementController.java

```diff
@@ -1,287 +0,0 @@
-package org.carlspring.strongbox.controllers;
```

Comment:
I don't think we should be dropping the controller completely, as there's functionality that the Spring Boot logging actuators don't have. For example, we have directory browsing, download entire logs, download config file, etc.

Could you please make an item list and check which of the methods have analogues in Spring Boot's implementation and which ones don't so that we could come up with an action plan?

Perhaps we could re-work our code and extend the Spring Boot Logging Actuator and contribute it back to the Springframework folks...?